app/persistence/fabric/MetricService.ts

```diff
@@ -701,7 +701,7 @@ export class MetricService {
 	getTxByOrgs(network_name: any, channel_genesis_hash: any) {
 		const sqlPerOrg = ` select count(creator_msp_id), creator_msp_id
       from transactions
-      where channel_genesis_hash =$1 and network_name=$2
+      where Trim(creator_msp_id) > '' and channel_genesis_hash =$1 and network_name=$2
```

Comment:
[nitpick]:  how about using `not equal` operator `<>`.

Reply:
yes we can use not equal but in order to handle the case trailing spaces along with empty fields used "TRIM" function.